src/content/en/2022/css.md

```diff
@@ -6,8 +6,9 @@ description: CSS chapter of the 2022 Web Almanac covering trends, changes, and p
 authors: [rachelandrew]
 reviewers: [j9t, meyerweb, svgeesus]
 analysts: [rviscomi]
-editors: [dereknahman]
+editors: [rviscomi]
 translators: []
+rachelandrew_bio: TODO
```

Comment:
@rachelandrew could you suggest a bio for your self? 2021 for reference: https://almanac.httparchive.org/en/2021/css#authors

Reply:
Added a placeholder bio for now